anaconda_anon_usage/utils.py

```diff
@@ -6,7 +6,9 @@
 from threading import RLock
 
 DPREFIX = os.environ.get("ANACONDA_ANON_USAGE_DEBUG_PREFIX") or ""
-DEBUG = bool(os.environ.get("ANACONDA_ANON_USAGE_DEBUG")) or DPREFIX
+DEBUG = (
```

Comment:
@mcg1969 is this what you had in mind?

Reply:
Not quite, we should be using `context.json`, IMO
I didn't appreciate you were working on a PR too. I've got one coming that has some tests baked in too. But I think we need to discuss how it should be have

Reply:
Sorry about that, I should have mentioned it! Won't happen again